### ISSUE TITLE
Huggingface "sentiment-analysis" pipeline always output "POSITIVE" label even for negative sentences

### ISSUE THREAD

**Issue by abmitra84** [huggingface/transformers#7010]
## Environment info
- `transformers` version: 3.1
- Platform: AWS , Colab etc.
- Python version: 3.6
- PyTorch version (GPU?): NA
- Tensorflow version (GPU?): NA
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

### Who can help

 examples/token-classification: @stefan-it

## Information

Model I am using (Bert: https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english):

The problem arises when using:
* [T ] the official example scripts: (give details below)
from transformers import pipeline
classifier = pipeline('sentiment-analysis')

classifier("I hate this thing") -->Returns Postive label
In fact when tested with a large bacth it never returned Negative label at all

* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [ ] my own task or dataset: (give details below)

## To reproduce

1. Get transformer 3.1
2. Use pipeline("sentiment-analysis")
3. Get label and score

## Expected behavior

from transformers import pipeline
classifier = pipeline('sentiment-analysis')

classifier("I hate this thing") -->Returns Negative label

Updates post some more experiments:

- I am using  distilbert-base-uncased-finetuned-sst-2-english following instruction from :https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english
- When I download the model and save it in local before using it through AutoModel and AutoTokenizer the score for sentence nlp("i dislike it") is 0.5453
- when I directly use it without manually downloading the model to local the same score jumps to 0.99 [same score in hosted modelhub inference api as well]
- another observation is that the tokenizer_config.json has both special_tokens_map_file and full_tokneizer_file as Null. Not sure if it supposed to be like this
- for my purpose I have to use the downloaded and manually loaded style of inferencing

**Comment by abmitra84** [comment]
Upon further updates, I think the issue was with using wrong AutoModel function. I replaced it with AutoModelforSequenceClassification at the time of download and it worked.
